<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>CAN Example</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QGroupBox" name="GB_messagesrecus">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="title">
       <string>Reception des trames CAN</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="L_InfosReception">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="text">
          <string>Temps écoulé        Drapeaux  ID   DLC  Données</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="TE_messagesrecus">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="undoRedoEnabled">
          <bool>false</bool>
         </property>
         <property name="lineWrapMode">
          <enum>QTextEdit::NoWrap</enum>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="L_statusBus">
           <property name="text">
            <string>Status du bus CAN :</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="L_statusCAN">
           <property name="text">
            <string>Inconnu</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="SendFrameBox" name="GB_messagesecrits">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="title">
       <string>Envoi des trames CAN</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="Barre_menu">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <widget class="QMenu" name="MenuAppels">
    <property name="geometry">
     <rect>
      <x>291</x>
      <y>115</y>
      <width>204</width>
      <height>205</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="toolTipDuration">
     <number>0</number>
    </property>
    <property name="title">
     <string>&amp;Appel</string>
    </property>
    <addaction name="AC_connecter"/>
    <addaction name="AC_deconnecter"/>
    <addaction name="separator"/>
    <addaction name="AC_resetcontroller"/>
    <addaction name="separator"/>
    <addaction name="AC_nettoyer"/>
    <addaction name="separator"/>
    <addaction name="AC_quitter"/>
   </widget>
   <widget class="QMenu" name="MenuAide">
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="toolTipDuration">
     <number>0</number>
    </property>
    <property name="title">
     <string>&amp;Aide</string>
    </property>
    <addaction name="actionPluginDocumentation"/>
   </widget>
   <widget class="QMenu" name="MenuFenetre">
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="toolTipDuration">
     <number>0</number>
    </property>
    <property name="title">
     <string>Fenêtre</string>
    </property>
    <addaction name="AC_afficherTwoDV"/>
   </widget>
   <addaction name="MenuAppels"/>
   <addaction name="MenuFenetre"/>
   <addaction name="MenuAide"/>
  </widget>
  <widget class="QToolBar" name="Barre_outils">
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="toolTipDuration">
    <number>0</number>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="AC_connecter"/>
   <addaction name="AC_deconnecter"/>
   <addaction name="separator"/>
   <addaction name="AC_nettoyer"/>
  </widget>
  <widget class="QStatusBar" name="Barre_de_status">
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <action name="AC_connecter">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/CANbus/Images/CANbus/Connect.png</normaloff>:/CANbus/Images/CANbus/Connect.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Connecter</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
     <kerning>true</kerning>
    </font>
   </property>
  </action>
  <action name="AC_deconnecter">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/CANbus/Images/CANbus/Disconnect.png</normaloff>:/CANbus/Images/CANbus/Disconnect.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Déconnecter</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="AC_quitter">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/CANbus/Images/CANbus/Exit.png</normaloff>:/CANbus/Images/CANbus/Exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quitter</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>&amp;About Qt</string>
   </property>
  </action>
  <action name="AC_nettoyer">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/CANbus/Images/CANbus/Clear.png</normaloff>:/CANbus/Images/CANbus/Clear.png</iconset>
   </property>
   <property name="text">
    <string>Nettoyer</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionPluginDocumentation">
   <property name="text">
    <string>Documentation PlugIn</string>
   </property>
   <property name="toolTip">
    <string>Open plugin documentation in Webbrowser</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="AC_resetcontroller">
   <property name="text">
    <string>&amp;Reset</string>
   </property>
   <property name="toolTip">
    <string>Reset CAN Controller</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="AC_afficherTwoDV">
   <property name="text">
    <string>Afficher TwoDV</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SendFrameBox</class>
   <extends>QGroupBox</extends>
   <header location="global">sendframebox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
